test/integration/search_test.rb

```diff
@@ -0,0 +1,34 @@
+require "test_helper"
+
+class ListSearchResultsTest < ActionDispatch::IntegrationTest
+  def setup
+    user = User.create!(username: "Victoria", email: "victoria@email.com", password: "secretpassword")
+    @article = Article.create!(title: "A title", description: "This is nice", user: user)
+    @article2 = Article.create!(title: "Another nice title", description: "This is really nice", user: user)
```

Comment:
Ich fände es gut, wenn hier in der description nicht das Wort nice vorkäme, weil wir dadurch beweisen würden, dass auch im title gesucht wird.

Reply:
Stimmt, mache ich